openfermionpyscf/_molecular_data.py

```diff
@@ -0,0 +1,123 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Class to use pyscf program to access quantum chemistry data."""
+
+from functools import reduce
+
+import numpy
+from openfermion.hamiltonians import MolecularData
+from pyscf import ao2mo
+
+
+class MolecularData(MolecularData):
+    def __init__(self, geometry=None, basis=None, multiplicity=None,
+                 charge=0, description="", filename="", data_directory=None):
+        super(MolecularData, self).__init__(
+            geometry, basis, multiplicity,
+            charge, description, filename, data_directory)
+        self._pyscf_data = {}
+
+    @property
+    def canonical_orbitals(self):
```

Comment:
For all of these you do not need full doc strings, but please add a one liner for each saying what the method does.